docs/sphinx_pyodide/sphinx_pyodide/packages.py

```diff
@@ -11,6 +11,13 @@
 
 from pyodide_build.io import parse_package_config
 
+DUMMY_PACKAGES = (
+    "cpp-exceptions-test",
```

Comment:
Maybe better to add a setting to `meta.yaml` to exclude a package from the docs?

Reply:
That's one other option, but I don't like adding a non-build-related section to `meta.yaml`.

Reply:
Maybe just under `package`? We could even use the version field:
```yaml
package:
  name: test-blah
  version: pyodide-test
```
We might also consider putting `version: python-stdlib` and `version: python-test` for the stdlib packages / test packages since none of these have real versions... Or better, leave the version out and put these as some sort of marker instead of a version...